aten/src/ATen/core/SingletonSymNodeImpl.h

```diff
@@ -85,6 +118,9 @@ class TORCH_API SingletonSymNodeImpl : public SymNodeImpl {
     if (coeff_ == 1) {
       return "j" + std::to_string(val_);
     }
+    if (type_ == SingletonVariant::CPP) {
+      return "jx";
```

Comment:
You're going to need to update this somehow (and the print above)

Reply:
No longer need to update this since we're no longer removing val.